File: app/src/main/java/com/arnaudpiroelle/manga/provider/mangapanda/api/MangaPandaDataApiService.java
package com.arnaudpiroelle.manga.provider.mangapanda.api;

import retrofit.client.Response;
import retrofit.http.GET;
import retrofit.http.Path;

public interface MangaPandaDataApiService {
    @GET("/{mangaAlias}/{chapterNumber}/{pageName}.{extension}")
    Response downloadPage(@Path("mangaAlias") String mangaAlias, @Path("chapterNumber") String chapterNumber,
                          @Path("pageName") String pageName, @Path("extension") String extension);
}

File: temp/mangapanda/api/MangaPandaDataApiService.kt
package com.arnaudpiroelle.manga.provider.mangapanda.api

import retrofit.client.Response
import retrofit.http.GET
import retrofit.http.Path

interface MangaPandaDataApiService {
    @GET("/{mangaAlias}/{chapterNumber}/{pageName}.{extension}")
    fun downloadPage(@Path("mangaAlias") mangaAlias: String, @Path("chapterNumber") chapterNumber: String,
                     @Path("pageName") pageName: String, @Path("extension") extension: String): Response
}
